README.md

```diff
@@ -1,2 +1,16 @@
 # ssis.bit
 AIO solution for SSIS students
+
+## Hardware
+
+CircuitPython supports [more than 200 different boards](https://circuitpython.org/downloads). Locally available is the [TTGO T8 ESP32-S2 ST7789](https://circuitpython.org/board/lilygo_ttgo_t8_s2_st7789/) at several online stores. The board includes the powerful 240MHz CPU, 4MB Flash and 8MB PSRAM, a 1.14" ST7789 display, a microSD card slot for virtually unlimited local data storage, battery connector 1.25mm JST with charge controller and regular USB-C connector. And buildin WiFi, of course.
+
+The planned adapterplate adds a 3-way button for input, place to hold the little 350 mAh battery and interface for I2C with regular 2.54mm 4pin JST XH as well as the small Stemma QT 1mm JST SH connector (QWIIC).
```

Comment:
Please include a picture

Reply:
The webpage has it.

Reply:
Without our daughter board it looks like this:
![T8](https://user-images.githubusercontent.com/96434905/147204458-49281430-32e1-4637-b3bd-fcda5f99e73c.jpg)

Reply:
Running CircuitPython it looks like this:
![lilygo_ttgo_t8_s2_st7789](https://user-images.githubusercontent.com/96434905/147204596-6282c74b-17ea-407d-ae82-2241ae201bc1.jpg)

Reply:
can you included it in the about page?